corecfg/refresh.go

```diff
@@ -20,6 +20,9 @@
 package corecfg
```

Comment:
Let's please not change this right now as it's an unrelated change that will move a lot of code around, and thus make this unrelated PR more confusing, but it feels like this package is misplaced at the top level. This is quite dependent on the whole infrastructure of the overlord being live, and also strongly tied to two pieces of it: the hook manager and the config manager. Something like "configstate/configcore" might be a more appropriate place for it (homage to the _io/ioutil_ and _path/filepath_ naming style).

Reply:
Indeed, that makes sense. I will work on this in a followup PR.